packages/react-magma-dom/src/components/ToggleButton/ToggleButton.test.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { axe } from '../../../axe-helper';
+import { ButtonSize } from '../Button';
+import { ToggleButton } from '.';
+import { fireEvent, render } from '@testing-library/react';
+import { SettingsIcon } from 'react-magma-icons';
+import { magma } from '../../theme/magma';
+import { transparentize } from 'polished';
+
+const TEXT = 'Test Text';
+
+const testId = 'test-id';
+
+const icon = <SettingsIcon />;
+
+describe('ToggleButton', () => {
+  it('Should find element by testId', () => {
+    const { getByTestId } = render(
+      <ToggleButton testId={testId}>{TEXT}</ToggleButton>
+    );
+
+    expect(getByTestId(testId)).toBeInTheDocument();
+  });
+
+  it('Icon only buttons are compliant with accessibility', () => {
+    const { container } = render(
+      <ToggleButton icon={icon} aria-label="Icon Button" />
+    );
+    return axe(container.innerHTML).then(result => {
+      return expect(result).toHaveNoViolations();
+    });
+  });
```

Comment:
Wouldn't hurt to also have an accessibility compliant test for non-icon toggle buttons